src/components/menu/Menu.jsx

```diff
@@ -19,25 +30,32 @@ const Menu = () => {
           </div>
         )}
 
+        {isOpen && <div className="overlay" onClick={closeMenu} />}
+        
         {isOpen && (
-          <>
-            <div className="overlay" onClick={closeMenu} />
-            <div className="side-menu">
-              <button className="close-button" onClick={closeMenu}>
-                &times; 
-              </button>
-              <h1 className='name'>Levantini</h1>
-              <ul className="menu-options">
-                <li><MdHome /> Home</li>
-                <li><LiaCrownSolid /> Leaderboard</li>
-                <li><IoIosLogOut /> Log out</li>
-              </ul>
-            </div>
-          </>
+          <div className="side-menu">
+            <button className="close-button" onClick={closeMenu}>
+              &times; 
+            </button>
+            <h1 className='name'>Levantini</h1>
+            <ul className="menu-options">
+              <li onClick={closeMenu} style={{ cursor: 'pointer' }}>
+                <Link to="/" style={{ textDecoration: 'none', color: 'inherit' }}>
+                  <MdHome /> Home
+                </Link>
+              </li>
+              <li><LiaCrownSolid /> Leaderboard</li>
+              <li style={{ display: 'flex', alignItems: 'center', cursor: 'pointer' }} onClick={logout}>
```

Comment:
we have two onClicks here. You can remove the li onClick and the LogoutGoogle onclick will bubble up to the li element